client/scss/components/_panel.scss

```diff
@@ -67,6 +65,7 @@ $header-button-size: theme('spacing.6');
   display: inline-grid;
   justify-content: center;
   align-content: center;
+  flex-shrink: 0;
```

Comment:
Technically a separate issue but more prevalent when text in the header area gets longer.